webapp/channels/src/components/menu/menu_item.tsx

```diff
@@ -188,8 +157,8 @@ export function MenuItem(props: Props) {
             isDestructive={isDestructive}
             hasSecondaryLabel={hasSecondaryLabel}
             isLabelsRowLayout={isLabelsRowLayout}
+            onClick={handleClick}
```

Comment:
Sweet it's always preferred to use I click I don't remember why I had to use mouse down probably because space key press was not being registered. Regardless onclick is great